### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by rabajaj0509**

### CODE REVIEW

**Comment by kushaldas** [comment]
Please rename the file to all small letters, and also squash your commits into one.

File: rebootAtleatOnce.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import unittest
+from testutils import system
```
Comment (trishnaguha) [inline rebootAtleatOnce.py]: We need to do relative import. We can use `from .testutils import system` instead.

**Comment by trishnaguha** [comment]
@rahulbajaj0509 We should have a 4 spaces indentation. Seems you are using tab key. Make sure your tab key is set to 4 spaces :).

**Comment by trishnaguha** [comment]
@rahulbajaj0509 You were working on the reboot test. But it seems you have git issue. journal log test wasn't part of it :). You need to fix it.

File: rebootatleastonce.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import unittest
+from .testutils import system
+
+class Reboot(unittest.TestCase):
```
Comment (trishnaguha) [inline rebootatleastonce.py]: Change the name to `TestReboot`

**Comment by trishnaguha** [comment]
@rahulbajaj0509 Follow PEP8 rule :)

**Pull request closed without merging**